src/python/pants/bin/pants_loader.py

```diff
@@ -49,7 +58,7 @@ def ensure_locale(cls):
     # libraries called by Pants may fail with more obscure errors.
     encoding = locale.getpreferredencoding()
     if encoding.lower() != 'utf-8' and os.environ.get(cls.ENCODING_IGNORE_ENV_VAR, None) is None:
-      raise cls.InvalidLocaleError(dedent("""\
+      raise cls.InvalidLocaleError(dedent("""
```

Comment:
I removed the `\` so that the error prints on a new line. Otherwise the formatting looks bad when printed.

This is what the two error messages in this file look like now, rather than the first line being on the same line as `__main__.InvalidInterpreter:` and messing up the indent:
```
__main__.InvalidInterpreter:
You are trying to run Pants with an unsupported Python interpreter. Pants
requires a Python 2.7 or a Python 3.6+ interpreter to be discoverable on
your PATH to run.

If you still get this error after ensuring at least one of these interpreters
is discoverable on your PATH, you may need to modify your build script
(e.g. `./pants`) to properly set up a virtual environment with the correct
interpreter. We recommend following our setup guide and using our setup script
as a starting point: https://www.pantsbuild.org/setup_repo.html.
```